index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Product Landing Page</title>
+        <meta charset="UTF-8" name="viewport" content="width=device, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header id="header">
+            <img id="header-img" src="360_F_339931799_lxc97IIbZ3ZeXYNPg3y15MIr8nCf1g8c.jpg" alt="Company Logo"><br>
+            <h1>Chawanzi & Dire Inc.</h1><br>
+            <nav id="nav-bar" >
+                <ul>
+                    <li class="nav-link"><a href="#">Home</a></li>
+                    <li class="nav-link"><a href="#">Gallery</a></li>
+                    <li class="nav-link"><a href="#">About Us</a></li>
+                </ul><br>
+                <video id ="video" width="" height="" controls>
+                    <source src="" type="">
```

Comment:
There should be valid values in the width, height, source and type parameters, please change that.